apps/content_filters/filter_condition/filter_condition_resource.py

```diff
@@ -75,3 +76,11 @@ class FilterConditionResource(Resource):
     privileges = {'POST': 'content_filters',
                   'PATCH': 'content_filters',
                   'DELETE': 'content_filters'}
+
+    def pre_request_get(self, request, lookup):
```

Comment:
imo this schema update is only needed for patch/post requests to do validation, or is there any other usecase?